src/lib/IlmThread/IlmThreadMutex.h

```diff
@@ -146,7 +70,7 @@ class ILMTHREAD_EXPORT Lock
         }
     }
     
-    ~Lock ()
+    inline ~Lock ()
```

Comment:
Aren't class methods with implementations right in the class declaration implicitly inline?

Reply:
yeah, I was originally trying to remove this entirely, then realized I couldn't and didn't rollback my attempt to make the functions declared later. Will kill those inline